packages/opentelemetry/src/asyncContextStrategy.ts

```diff
@@ -30,7 +31,7 @@ export function setOpenTelemetryContextAsyncContextStrategy(): void {
     const ctx = api.context.active();
 
     // We depend on the otelContextManager to handle the context/hub
-    return api.context.with(ctx, () => {
+    return api.context.with(ctx.setValue(SENTRY_FORK_ISOLATION_SCOPE_CONTEXT_KEY, true), () => {
```

Comment:
we use this attribute to tell the otel context manager if the isolation scope should be forked or not.

Reply:
Can we explain this a bit further in comments? I still don't fully get it.

Reply:
will do!